tests/test_ops_regrid.py

```diff
@@ -1,20 +1,22 @@
 import os
+import sys
 from pathlib import Path
 
 import cf_xarray  # noqa
 import pytest
 import xarray as xr
 from roocs_grids import get_grid_file, grid_dict
 
+from _common import ATLAS_v1_CORDEX  # noqa
+from _common import ATLAS_v1_EOBS_GRID  # noqa
```

Comment:
Why was it necessary to import these two dataset specification separately?

Reply:
I had added a `noqa` to both in order to prevent autoformatters from accidentally removing them (they aren't explicitly called in code). `isort` separated them for some reason. I'll at least put these on the same line.